### PR TITLE
Add PieChart and DonutChart description

### PR DESCRIPTION
**Pull request by kandl**
- Provide high-level description of the following Pluggable charts (buckets, dimesions, sorts):
    * DonutChart
    * PieChart

JIRA: RAIL-4052

---

Supported PR commands:

| Command                  | Description            |
| ------------------------ | ---------------------- |
| `ok to test`             | Re-run standard checks |
| `extended test`          | BackstopJS tests       |
| `extended check sonar`   | SonarQube tests        |
| `extended check cypress` | Cypress E2E tests      |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `check-extended-cypress` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/7001/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/8867/ : SUCCESS in 10m 58s

File: libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/donutChart/PluggableDonutChart.tsx [hunk 1]
```diff
@@ -1,10 +1,44 @@
...
+ *
+ * | Name     | Id       | Accepts             |
+ * |----------|----------|---------------------|
+ * | Measures | measures | measures only       |
+ * | ViewBy   | view     | attribute or date   |
+ *
+ * ### Bucket axioms
+ *
+ * - |ViewBy| ≤ 1
+ * - |Measures| ≥ 1
```
Comment (no23reason) [inline libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/donutChart/PluggableDonutChart.tsx:21]: What is the maximum nr of Measures?

File: libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/pieChart/PluggablePieChart.tsx [hunk 2]
```diff
@@ -41,6 +41,40 @@ import cloneDeep from "lodash/cloneDeep";
...
+ *
+ * | Name     | Id       | Accepts             |
+ * |----------|----------|---------------------|
+ * | Measures | measures | measures only       |
+ * | ViewBy   | view     | attribute or date   |
+ *
+ * ### Bucket axioms
+ *
+ * - |ViewBy| ≤ 1
+ * - |Measures| ≥ 1
```
Comment (no23reason) [inline libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/pieChart/PluggablePieChart.tsx:57]: What is the maximum nr of Measures?

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/7065/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/8916/ : SUCCESS in 10m 40s

**Review by no23reason — APPROVED** [review]

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/2045/ : SUCCESS in 58s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/2328/ : SUCCESS in 19s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/1141/ : SUCCESS in 13m 36s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/8917/ : SUCCESS in 11m 01s

**Pull request merged**